SU2_PY/SU2/opt/project.py

```diff
@@ -368,8 +375,14 @@ def init_design(self,config,closest=None):
         # update local state filenames ( ??? why not in Design() )
         for key in design.files:
             name = design.files[key]
-            name = os.path.split(name)[-1]
-            design.files[key] = name
+            if isinstance(name,list):
+            	built_name = []
```

Comment:
double check indentation throughout

Reply:
Yup, I made the mistake of editing some of these files in vim and using tabs. Fixed